### DISCUSSION TITLE
How long does it usually take to get a pull-request reviewed and approved?

### DISCUSSION THREAD

**Discussion by magroski** [Codeception/Codeception#6087] [Q&A] [answered]
My team found an issue with codeception-phalcon and Phalcon 3.4.
Phalcon Cookie object accepts null as default value for the secure attribute, which is acceptable since cookie attributes are treated as false by the browser when they aren't set.
However, codeception's Phalcon connector only accepts true|false and is breaking when a cookie is created using default values.
It's a one-line fixer that has been sent here: https://github.com/Codeception/module-phalcon/pull/6

**Comment by Naktibalda** [comment, marked as answer]
I released it as 1.1.0, because there was an unpublished year old change in https://github.com/Codeception/module-phalcon/pull/3 .

Does this secure cookie issue exist in Phalcon4? 
https://github.com/Codeception/module-phalcon4/blob/master/src/Codeception/Lib/Connector/Phalcon4.php#L146 contains very similar code.
